app/src/main/java/com/pajato/android/gamechat/common/InvitationManager.java

```diff
@@ -96,4 +246,97 @@ private boolean hasGroupInvite(@NonNull final Account account, @NonNull final St
         return true;
     }
 
+    /** Extend an invitation to join GameChat using AppInviteInvitation Intent */
+    public void extendAppInvitation(final FragmentActivity fragmentActivity) {
+        String firebaseUrl = FirebaseDatabase.getInstance().getReference().toString();
+        String dynamicLink = new Uri.Builder()
+                .scheme("https")
+                .authority(APP_CODE + ".app.goo.gl")
+                .path("/")
+                .appendQueryParameter("link", firebaseUrl)
+                .appendQueryParameter("apn", APP_PACKAGE_NAME)
+                .appendQueryParameter("afl", PLAY_STORE_LINK)
+                .appendQueryParameter("ifl", WEB_LINK)
+                .toString();
+
+        Log.i(TAG, "dynamicLink=" + dynamicLink);
+        Intent intent = new AppInviteInvitation.IntentBuilder(fragmentActivity.getString(R.string.InviteTitle))
+                .setMessage(fragmentActivity.getString(R.string.InviteMessage))
+                .setDeepLink(Uri.parse(dynamicLink))
+                .build();
+        fragmentActivity.startActivityForResult(intent, MainActivity.RC_INVITE);
+    }
+
+    /** Extend an invitation to join GameChat using AppInviteInvitation and specify a group to join. */
+    public void extendAppInvitation(final FragmentActivity fragmentActivity, final String groupKey) {
+        String firebaseUrl = FirebaseDatabase.getInstance().getReference().toString();
```

Comment:
I have a slight preference for using String.format(...) to build path references so that the code is clear and concise.  More of a style nit than a convention, however.